curriculum/challenges/english/25-front-end-development/lab-magazine-layout/6735a7370e0ae93a4577c689.md

```diff
@@ -0,0 +1,263 @@
+---
+id: 6735a7370e0ae93a4577c689
+title: Design a Magazine Layout
+challengeType: 25
+dashedName: design-a-magazine-layout
+demoType: onClick
+---
+
+# --description--
+
+In this lab, you will design a magazine layout using CSS Grid, including concepts like grid rows and grid columns.
+
+**Objective:** Fulfill the user stories below and get all the tests to pass to complete the lab. 
+
+**User Stories:**
+
+1. You should have a `main` element with the class `magazine-cover`.
+
+2. Your `.magazine-cover` should contain a `header` element for the title with the class `title`.
+
+3. You should have four `section` elements in your page. The first three sections for magazine articles and a fourth `section` element for a cover image.  
+
+4. The four `section` elements should have a `class` attribute of `feature-article`, `small-article1`, `small-article2`, and `cover-image`, respectively.
+
+5. You should set the `display` property of `.magazine-cover` to `grid.`
+
+6. The `.magazine-cover` should have a `grid-template-areas` property that defines the layout of the grid:
```

Comment:
From this instruction, I understood that there should be 2 featured posts in the second row. However, the solution/ demo asks for only one.

This is the layout I came up with and it doesn't pass:

<details>

<summary>Image</summary>

![image](https://github.com/user-attachments/assets/de0e3acc-6b8f-43c3-b0d5-be3ff3618f9c)

</details>

Reply:
Oh you're right. Maybe it should say `A feature article spanning two columns and a cover image in the second row`

Reply:
Yes, that would make the instruction clear.

Reply:
I left two suggestions. If those are good I think we can go ahead so we merge this PR @zairahira